dkpro-core-maltparser-asl/src/scripts/build.xml

```diff
@@ -309,8 +309,8 @@
 				<entry key="DC.creator" value="Universitat Pompeu Fabra. Institut Universitari de Lingüística Aplicada (IULA)"/>
 				<entry key="DC.identifier" value="http://hdl.handle.net/10230/20407"/>
 				<entry key="DC.rights" value="http://creativecommons.org/licenses/by/3.0/"/>
-	    		<entry key="pos.tagset" value="freeling"/>
-				<entry key="dependency.tagset" value="iula"/>
+	    		<entry key="pos.tagset" value="ancora"/>

```

Comment:
When changing the metadata, the metadata version needs to be increased as well - here and also in the respective dependency in the pom.xml.